### PR TITLE
Add Bankera

### PR DESCRIPTION
**Pull request by 40504**

### CODE REVIEW

File: img/b/bankera.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...6.8.2-.6Zm11.4 3.5.6 8v.5H67l-9-4h-.2c0-.4 0-.4.4-.4l8.2-4.5h.2v.4h.2Zm-13.1 4.7L38.4 81H38v-.2l2.8-5.7.2-.4 12.6 2.1v.6Zm-38-39.9 7.5-4.7h.4l1.7 7.5v.4h-.4l-9.2-3.2-.2.2c0-.2 0-.2.2-.2Zm7.9 13.6-4-3.7V47l5.3-4.6h.4v.4L23.7 51l-.2.2Zm8.2-.8-3.3 8-.4.5-.4-.4-3.4-6c0-.3 0-.3.4-.3l6.5-2.2h.4v.4h.2Zm-7.5 3 3.4 5.8v.4l-8 4.2H19v-.4L23.7 53c.2 0 .2-.2.3-.2l.2.4Zm0 22.7 4-14.3h.4L32 73.5c0 .4 0 .4-.4.4l-7 2.7h-.5V76Zm6.5-.5-6.3 6.8H24l-.1-.2v-4.5l.1-.4 6.3-2c.4-.3.4 0 .4 0s.4 0 0 .3Zm5.7 6.4-3 5.7s-.3.4-.3 0h-.4V76c0-.2.2-.4.4-.4v.4l3.3 5.7v.3ZM32 89l-7-5.2c-.3 0-.3-.2-.5-.2l7.5-8h.4v.4l.2 12.7c0 .4 0 .4-.2.4l-.4-.2Zm-8.4-10.3v4.5c0 .4 0 .4-.4.4l-7.3 3.6c0 .3 0 0-.4 0v-.2l7-8.1c0-.2.3-.2.3-.4l.8.2Zm32.8 8L39.2 82v-.3l16.7-4h.4l.4 8.7v.6l-.4-.2Zm-36.6-22 7.5-3.9h.2v.2l-4 14.2v.2c-.1 0-.1-.2-.3-.2l-3.6-10.3.2-.1Zm12-49.3L28.4 23H28L21 17v-.8L31.3 15h.4v.4ZM22 23.2l-1.6 1.2-.4-.2.4-4.2v-.3c.2 0 .2 0 .2.3l1.2 3v.2h.2Zm-.4-4.7 5.1 4.7v.4h-.4l-3.3-.8-1.6-4.1c-.4 0 0-.4 0-.4l.2.2Zm6 6.3-3.3 5.7h-.5l-1-6.3V24h.2l4.5.6v.2h.2Zm4 8.8-6.9-1.8c-.2-.4-.2-.4 0-.4l3.6-6.2h.6l3 8c0 .4 0 .4-.3.4Zm-9-4.5L20 25.7c0-.4 0-.4.4-.4l1.6-1.5.4.2.6 4.7v.4c-.6.2-.6.2-.6 0Z" fill="#46BAB6" fill-rule="nonzero"/></svg>
```
Comment (phallobst) [inline img/b/bankera.com.svg]: The logo looks a bit rugged. How about this one:

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="45.23" height="45.23"><path d="M15.99 15.26V8.73a.1.1 0 0 0-.18-.04l-1.42 3.03a.1.1 0 0 0 0 .07l1.42 3.51a.1.1 0 0 0 .18-.04zm21.5 16.65-2.86 3.99a.1.1 0 0 0 .06.15l2.27.41a.1.1 0 0 0 .12-.07l.58-4.42a.1.1 0 0 0-.17-.06zm-2.56 2.58 2.26-3.15a.1.1 0 0 0-.1-.14l-1.6.36a.1.1 0 0 0-.08.07l-.65 2.78a.1.1 0 0 0 .17.08zm-8.6-25.04 2.98 1.45a.1.1 0 0 0 .14-.07l1.25-6.35a.1.1 0 0 0-.17-.08L26.3 9.3a.1.1 0 0 0 .03.15zM9.14 3.89 13.6.18a.1.1 0 0 0-.06-.16L7.7.17a.1.1 0 0 0-.09.13L9 3.85a.1.1 0 0 0 .15.03zm-.48 1.14-1.06.9a.1.1 0 0 0-.03.07v1a.1.1 0 0 0 .07.09l1.76.48a.1.1 0 0 0 .11-.12l-.7-2.38a.1.1 0 0 0-.15-.04zm5.77-.38-5 .06a.1.1 0 0 0-.09.13l.77 2.62a.1.1 0 0 0 .14.06l4.24-2.7a.1.1 0 0 0-.06-.17zM10 4.14l5.3-.07a.1.1 0 0 0 .08-.12L14.5.37a.1.1 0 0 0-.15-.05l-4.4 3.65a.1.1 0 0 0 .05.17zm5.45.75L11.7 7.28a.1.1 0 0 0 .07.17l4.08-.65a.1.1 0 0 0 .07-.11l-.31-1.73a.1.1 0 0 0-.15-.07zm.59-1.3L18.98.17A.1.1 0 0 0 18.9 0h-3.78a.1.1 0 0 0-.09.12l.84 3.44a.1.1 0 0 0 .17.04zm.73.3 3.48-.62a.1.1 0 0 0 .08-.12l-.7-2.66a.1.1 0 0 0-.16-.04l-2.79 3.28a.1.1 0 0 0 .09.15zm0 .74 2.68 1.68a.1.1 0 0 0 .15-.05l.66-2.27a.1.1 0 0 0-.1-.13l-3.36.6a.1.1 0 0 0-.03.17zm1.79 1.8-2.2-1.38a.1.1 0 0 0-.14.1l.27 1.49a.1.1 0 0 0 .1.08l1.93-.12a.1.1 0 0 0 .04-.17zm-11 1.36v2.28a.1.1 0 0 0 .17.06L9.35 8.3a.1.1 0 0 0-.04-.16L7.69 7.7a.1.1 0 0 0-.12.1zm1.86 1.3L7.6 11.14a.1.1 0 0 0-.02.07v.85a.1.1 0 0 0 .08.1l1.55.15a.1.1 0 0 0 .1-.09l.28-3.08a.1.1 0 0 0-.16-.07zm1.33 2.12-.47-1.39a.1.1 0 0 0-.18.02l-.18 1.92a.1.1 0 0 0 .16.08l.64-.53a.1.1 0 0 0 .03-.1zm.63-.1 1.51.2a.1.1 0 0 0 .08-.17l-2.1-1.9a.1.1 0 0 0-.15.1l.58 1.71a.1.1 0 0 0 .08.06zm-.63-2.76 3.11 2.83a.1.1 0 0 0 .15-.03l1.7-3.6a.1.1 0 0 0-.1-.13l-4.8.77a.1.1 0 0 0-.06.16zm2.72 3.6-1.89-.24a.1.1 0 0 0-.1.11l.47 2.99a.1.1 0 0 0 .18.02l1.42-2.74a.1.1 0 0 0-.08-.13zm-1.09 3.9 3.22.9a.1.1 0 0 0 .11-.13l-1.61-4a.1.1 0 0 0-.18 0l-1.6 3.1a.1.1 0 0 0 .07.13zM10.79 12l-.77.63a.1.1 0 0 0-.02.13l1.14 1.66a.1.1 0 0 0 .17-.07l-.37-2.3a.1.1 0 0 0-.15-.05zm-1.22 1.41L7.9 17.77a.1.1 0 0 0 .14.12l3.32-1.94a.1.1 0 0 0 .03-.13L9.73 13.4a.1.1 0 0 0-.17.02zm-2-.59v3.77a.1.1 0 0 0 .2.03L9.1 13a.1.1 0 0 0-.08-.13l-1.36-.13a.1.1 0 0 0-.1.1zm4 3.67L8.1 18.52a.1.1 0 0 0 .01.17l4.27 1.55a.1.1 0 0 0 .12-.12l-.8-3.55a.1.1 0 0 0-.15-.07zm.66 4.29-4.26-1.55a.1.1 0 0 0-.12.13l1.58 3.83a.1.1 0 0 0 .15.04l2.68-2.29a.1.1 0 0 0-.03-.16zm1-.76 2.4-2.54a.1.1 0 0 0-.04-.16l-3.19-.89a.1.1 0 0 0-.11.12l.78 3.42a.1.1 0 0 0 .16.05zm-5.65.59v5.96a.1.1 0 0 0 .18.04l1.28-2.8a.1.1 0 0 0 0-.07l-1.28-3.17a.1.1 0 0 0-.18.04zm6.2-.1 6.78.45a.1.1 0 0 0 .06-.17l-4.24-3.14a.1.1 0 0 0-.13 0l-2.54 2.7a.1.1 0 0 0 .07.15zm-3.91 3.36 1.7 1.62a.1.1 0 0 0 .15-.05l.73-3.69a.1.1 0 0 0-.15-.09l-2.43 2.07a.1.1 0 0 0 0 .14zm2.53 1.83 3.25-1.03a.1.1 0 0 0 .05-.16l-2.46-2.98a.1.1 0 0 0-.17.05l-.8 4.01a.1.1 0 0 0 .13.11zm-3-1.31-1.81 3.95a.1.1 0 0 0-.01.04v.7a.1.1 0 0 0 .15.08l3.62-2.94a.1.1 0 0 0 0-.14l-1.8-1.72a.1.1 0 0 0-.15.03zm2.95 2.05 1.64 2.93a.1.1 0 0 0 .18 0l1.55-3.96a.1.1 0 0 0-.12-.13l-3.2 1.02a.1.1 0 0 0-.05.14zm1.34 3.56-1.69-3a.1.1 0 0 0-.17.01l-1.9 4.7a.1.1 0 0 0 .13.12l3.58-1.7a.1.1 0 0 0 .05-.13zm-2.79-2.68L7.6 30a.1.1 0 0 0-.03.07v.86a.1.1 0 0 0 .04.08l1.46 1a.1.1 0 0 0 .14-.04l1.83-4.53a.1.1 0 0 0-.15-.11zm5.09 2.72v-3.26a.1.1 0 0 0-.18-.03l-1.22 3.1a.1.1 0 0 0 .07.14l1.22.14a.1.1 0 0 0 .11-.1zm-.06 4.85.05-4.09a.1.1 0 0 0-.08-.1l-1.23-.14a.1.1 0 0 0-.1.12l1.18 4.23a.1.1 0 0 0 .18-.02zm-3.6 3.61 3.41-1.1a.1.1 0 0 0 .07-.12l-1.58-5.66a.1.1 0 0 0-.18 0l-1.85 6.76a.1.1 0 0 0 .12.12zm1.18-7.67-3.76 1.78a.1.1 0 0 0-.05.12l1.9 5.05a.1.1 0 0 0 .17 0l1.87-6.84a.1.1 0 0 0-.13-.11zm-5.93 1.01v3.85a.1.1 0 0 0 .18.04l1.17-3.05a.1.1 0 0 0-.03-.11l-1.17-.8a.1.1 0 0 0-.15.07zm3.62 6.54-1.82-4.85a.1.1 0 0 0-.17 0l-1.54 3.9a.1.1 0 0 0 .07.14l3.35.94a.1.1 0 0 0 .11-.13zm-3.62-.15v5.9a.1.1 0 0 0 .17.05l3.44-4.92a.1.1 0 0 0-.05-.15l-3.44-.98a.1.1 0 0 0-.12.1zm3.72 1.88L8.62 43.9a.1.1 0 0 0 .13.14l2.7-1.75a.1.1 0 0 0 .05-.08l-.03-2.03a.1.1 0 0 0-.18-.06zm.43 2.67-3.27 2.12a.1.1 0 0 0 .05.18l6.51.14a.1.1 0 0 0 .06-.18l-3.24-2.26a.1.1 0 0 0-.1 0zm.3-3.51.03 2.16a.1.1 0 0 0 .17.06L15 38.43a.1.1 0 0 0-.1-.15l-2.82.9a.1.1 0 0 0-.06.1zm3.95 5.52-.07-6.26a.1.1 0 0 0-.17-.06l-3.45 3.8a.1.1 0 0 0 .02.14l3.52 2.46a.1.1 0 0 0 .15-.08zm2.34-3.53-1.66-2.48a.1.1 0 0 0-.18.05l.06 5.3a.1.1 0 0 0 .18.04l1.6-2.8a.1.1 0 0 0 0-.11zm1.49 3.78 3.17-.24a.1.1 0 0 0 .05-.17l-4.16-2.86a.1.1 0 0 0-.13.03l-1.87 3.26a.1.1 0 0 0 .1.14zm-2.96-7 1.72 2.55a.1.1 0 0 0 .16-.01l1.17-2.33a.1.1 0 0 0-.08-.14l-2.88-.2a.1.1 0 0 0-.09.14zm9.64.95-5.85-.8a.1.1 0 0 0-.1.04l-1.23 2.47a.1.1 0 0 0 .1.13l7.1-1.65a.1.1 0 0 0-.02-.19zm1.91 4.52-.1-4.05a.1.1 0 0 0-.12-.09l-7.95 1.86a.1.1 0 0 0 0 .19l8.06 2.19a.1.1 0 0 0 .11-.1zm-4.21 1.18 2.91-.63a.1.1 0 0 0 0-.18l-6.31-1.72a.1.1 0 0 0-.08.17l3.4 2.34c.02.02.05.02.08.02zm.41-7.51-2.13.5a.1.1 0 0 0 .01.18l4.07.57a.1.1 0 0 0 .06-.18l-1.94-1.06a.1.1 0 0 0-.07-.01zm.76-.27 2.59 1.41a.1.1 0 0 0 .14-.1l-.67-2.76a.1.1 0 0 0-.15-.05l-1.92 1.34a.1.1 0 0 0 0 .16zm3.51 2.65.1 3.88a.1.1 0 0 0 .14.08l4.21-2.16a.1.1 0 0 0 0-.18l-4.32-1.71a.1.1 0 0 0-.13.09zm4.98 1.08-.07-3.82a.1.1 0 0 0-.14-.08l-4.19 2.12a.1.1 0 0 0 .01.17l4.26 1.7a.1.1 0 0 0 .13-.09zm-4.98-2.12 4.08-2.08a.1.1 0 0 0-.02-.17l-4.84-1.01a.1.1 0 0 0-.1.11l.74 3.08a.1.1 0 0 0 .14.07zm5.48-1.85.07 3.6a.1.1 0 0 0 .16.06 14.77 14.77 0 0 0 2.25-3.2.1.1 0 0 0-.07-.13l-2.3-.43a.1.1 0 0 0-.11.1zm-6.35-2.02 4.87 1.01a.1.1 0 0 0 .09-.16l-4.17-4.2a.1.1 0 0 0-.16.04l-.7 3.2a.1.1 0 0 0 .07.1zm1.45-3.49 4.32 4.36a.1.1 0 0 0 .16-.05l.9-3.8a.1.1 0 0 0-.1-.11l-5.2-.56a.1.1 0 0 0-.08.16zm6.27-.25 1.87-.42a.1.1 0 0 0 .07-.11 27.42 27.42 0 0 0-1.01-3.44.1.1 0 0 0-.19 0l-.85 3.85a.1.1 0 0 0 .11.12zm-2.67-.3-5.23-2.49a.1.1 0 0 0-.13.13l.97 2.03a.1.1 0 0 0 .08.06l4.26.46a.1.1 0 0 0 .05-.18zM27.6 27.4l6.5 3.1a.1.1 0 0 0 .12-.14l-3.64-5.63a.1.1 0 0 0-.14-.02l-2.86 2.54a.1.1 0 0 0 .02.15zm3.74-2.55 3.6 5.56a.1.1 0 0 0 .17-.03l.99-4.44a.1.1 0 0 0-.07-.11l-4.59-1.12a.1.1 0 0 0-.1.14zm2.02-2.47-2 1.57a.1.1 0 0 0 .04.16l4.08 1a.1.1 0 0 0 .1-.15l-2.09-2.57a.1.1 0 0 0-.13 0zm-2.5 1.23 2.03-1.58a.1.1 0 0 0 0-.16c-.87-.57-1.8-1.03-2.78-1.4a.1.1 0 0 0-.12.11l.73 2.98a.1.1 0 0 0 .14.05zm-7.17.89 4.98-.42a.1.1 0 0 0 .02-.19l-6.12-2a.1.1 0 0 0-.11.13l1.13 2.42a.1.1 0 0 0 .1.06zm5.77.09-4.98.41a.1.1 0 0 0-.05.18L26.88 27a.1.1 0 0 0 .12 0l2.53-2.25a.1.1 0 0 0-.07-.16zm-7.85-2.77-4.06 2.5a.1.1 0 0 0 .05.17l5.25.02a.1.1 0 0 0 .08-.13l-1.18-2.53a.1.1 0 0 0-.14-.03zm-5.21 2.54 4.26-2.63a.1.1 0 0 0-.04-.18l-6.82-.46a.1.1 0 0 0-.08.15l2.56 3.1a.1.1 0 0 0 .12.02zm4.34-6.85h-3.32a.1.1 0 0 0-.06.17l3.86 2.86a.1.1 0 0 0 .15-.1l-.53-2.85a.1.1 0 0 0-.1-.08zm2.6 4.04 6.67 2.17a.1.1 0 0 0 .12-.11l-.8-3.24a.1.1 0 0 0-.1-.07l-5.87 1.06a.1.1 0 0 0-.01.18zm-1.2-.55 5.84-1.06a.1.1 0 0 0 .01-.18l-6.4-2.07a.1.1 0 0 0-.13.11l.58 3.12a.1.1 0 0 0 .1.08zm-.18-3.79 5.62 1.81a.1.1 0 0 0 .09-.16l-4.2-3.3a.1.1 0 0 0-.14.03 5.2 5.2 0 0 1-1.4 1.46.1.1 0 0 0 .03.16zm2.65-1.47 4.16 3.25a.1.1 0 0 0 .15-.1l-.81-2.76a.1.1 0 0 0-.08-.07l-3.35-.49a.1.1 0 0 0-.07.17zm7.35.79-3.1-.37a.1.1 0 0 0-.1.12l.85 2.93a.1.1 0 0 0 .16.03l2.25-2.55a.1.1 0 0 0-.06-.16zm-6.29-6.03a33.75 33.75 0 0 1-1.81 4.24.1.1 0 0 0 .07.14l3.76.55a.1.1 0 0 0 .1-.13l-1.94-4.8a.1.1 0 0 0-.18 0zm3.12 5.08 3.17.37a.1.1 0 0 0 .1-.14l-2.22-3.57a.1.1 0 0 0-.18.02l-.94 3.2a.1.1 0 0 0 .07.12zm.42-4.1-2.69-1.3a.1.1 0 0 0-.13.13l1.78 4.39a.1.1 0 0 0 .18-.01l.91-3.09a.1.1 0 0 0-.05-.12zm1.07-1.04 3.04-2.6a.1.1 0 0 0 .03-.1 14.56 14.56 0 0 0-1.82-3.7.1.1 0 0 0-.17.03l-1.24 6.28a.1.1 0 0 0 .16.09zm3.43 1.92-3.3-.57a.1.1 0 0 0-.1.15l2.29 3.67a.1.1 0 0 0 .17 0c.5-1 .84-2.05 1.01-3.15a.1.1 0 0 0-.07-.1zm-.29-3.86-2.99 2.56a.1.1 0 0 0 .05.16l3.27.56a.1.1 0 0 0 .11-.08 10.76 10.76 0 0 0-.28-3.15.1.1 0 0 0-.16-.05zM25.81.57A32.66 32.66 0 0 0 20.23 0a.1.1 0 0 0-.1.12l1.7 6.57a.1.1 0 0 0 .17.03l3.87-6a.1.1 0 0 0-.06-.15zm-5.22 4.32-.48 1.63a.1.1 0 0 0 .07.12c.33.1.66.21.98.34a.1.1 0 0 0 .13-.1l-.51-2a.1.1 0 0 0-.19 0zm9.76-2.12a13.65 13.65 0 0 0-3.74-2 .1.1 0 0 0-.12.04l-1.64 2.56a.1.1 0 0 0 .09.14l5.36-.57a.1.1 0 0 0 .05-.17zm-4.35 6 4.41-5.1a.1.1 0 0 0-.08-.16l-5.68.6a.1.1 0 0 0-.08.12l1.27 4.5a.1.1 0 0 0 .16.04zm-2.26-.32.1-3.2a.1.1 0 0 0-.17-.06l-1.42 2.2a.1.1 0 0 0 .04.14c.48.25.92.58 1.29.98a.1.1 0 0 0 .16-.06zm.64-2.09-.09 2.78a.1.1 0 0 0 .1.1l.87-.02a.1.1 0 0 0 .09-.12l-.78-2.76a.1.1 0 0 0-.19.02zm.81 3.43-.58.01a.1.1 0 0 0-.09.14c.11.25.2.5.28.76a.1.1 0 0 0 .18 0l.3-.78a.1.1 0 0 0-.09-.13zM8.56 4.29l-.8-2.3a.1.1 0 0 0-.2.04v3a.1.1 0 0 0 .17.07l.8-.7a.1.1 0 0 0 .03-.11z" fill="#19a1a3"/></svg>
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Comment by moviuro** [comment]
@40504 I have serious doubts about if Bankera *truly* supports 2FA. Can you please confirm what happens when you:

1. login to your account through a private/incognito window
2. close all private/incognito windows
3. login to your account through a second private/incognito window

If you aren't asked for the 2FA code **twice**, then it's not 2FA.

I'm having very serious doubts because my own 2 banking accounts in France (HelloBank! and LCL) *don't* do it properly.

**Comment by 40504** [comment]
It's worked as is described here: https://bankera.com/faq/how-to-enable-second-factor-authentication/ checked right now with disposable number

**Comment by 40504** [comment]
> @40504 I have serious doubts about if Bankera _truly_ supports 2FA. Can you please confirm what happens when you:
> 
> 1. login to your account through a private/incognito window
> 2. close all private/incognito windows
> 3. login to your account through a second private/incognito window
> 
> If you aren't asked for the 2FA code **twice**, then it's not 2FA.
> 
> I'm having very serious doubts because my own 2 banking accounts in France (HelloBank! and LCL) _don't_ do it properly.

It's worked as is described here: https://bankera.com/faq/how-to-enable-second-factor-authentication/ checked right now with disposable number

**Pull request merged**